app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  has_many :microposts, dependent: :destroy
```

Comment:
has_many có tác dụng gì em

Reply:
Dạ has_many này là mình đang khai báo phía còn lại của relation của User với Micropost ạ, trong trường hợp này là quan hệ one to many ạ.